packages/fetch/src/fetch.ts

```diff
@@ -2,6 +2,8 @@ import { requestRetryLogger } from '@pnpm/core-loggers'
 import { operation, RetryTimeoutOptions } from '@zkochan/retry'
 import fetch, { Request, RequestInit as NodeRequestInit, Response } from 'node-fetch'
 
+export { isRedirect } from 'node-fetch'
```

Comment:
Small fix here to properly export `isRedirect`. I think problem was being masked by the babel -> cjs -> bundle with esbuild pipeline. When I started bundling directly with esbuild I found runtime errors cause `fetch.isRedirect` is [technically undefined](https://github.com/node-fetch/node-fetch/blob/c33e393c479392ff201c9c3b69bc2d18b8d88a23/src/index.js#L36) in the ES module output of node-fetch.